### PR TITLE
manage order in admin

### PR DESCRIPTION
**Pull request by HoaiPEC**
## Related Tickets:
- [27407](https://edu-redmine.sun-asterisk.vn/issues/27407)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by HoaiPEC** [comment]
ready

File: resources/lang/en/templates.php [hunk 1]
```diff
@@ -38,6 +38,15 @@
     'list_post' => 'List post',
     'login_admin' => 'Login Admin',
     'sign_in' => 'Sign in',
+    'order' => 'Order',
+    'infor_order' => 'Order information',
```
Comment (huyenttm-0901) [inline resources/lang/en/templates.php:42]: viết file vi đi

**Comment by HoaiPEC** [comment]
ready

File: resources/lang/vi/templates.php [hunk 2]
```diff
@@ -39,6 +39,8 @@
     'login_admin' => 'Login Admin',
     'sign_in' => 'Sign in',
     'import_bill' => 'Hóa đơn nhập hàng',
+    'order' => 'Đơn đặt hàng',
+    'infor_order' => 'Thông tin đơn đặt hàng',
     'list_import_bill' => 'Danh sách hóa đơn nhập hàng',
     'import_date' => 'Ngày nhập',
     'edit_import bill' => 'Sửa hóa đơn nhập hàng',
```
Comment (huyenttm-0901) [inline resources/lang/vi/templates.php:46]: sao c thấy chưa đủ key?

**Comment by HoaiPEC** [comment]
ready

**Pull request merged**